utils/sed/wrapper.py

```
@@ -0,0 +1,8 @@
+__author__ = "Filipe G. Vieira"
+__copyright__ = "Copyright 2026, Filipe G. Vieira"
+__license__ = "MIT"
+
+log = snakemake.log_fmt_shell(stdout=False, stderr=True)
```

Ruff (0.15.10)

[error] Undefined name `snakemake`

(F821)